### ISSUE TITLE
Transition program from V9.1 to V9.2 processes Table:TwoIndependentVariables incorrectly

### ISSUE THREAD

**Issue by jcyuan2020** [NREL/EnergyPlus#8281]
Issue overview
--------------
When a Table:TwoIndepenentVariables object exist in the idf file, the transition program will not recognize and convert it correctly. The transition program will try to use two Table:IndependentVariable objects, a Table:IndependentVariableList, and a Table:Lookup object to replace the original Table:TwoIndependentVariables obeject, but reporting some values that do not seem to confide with the original table. The following warning message will be thrown in the xxx_transition.audit file: 
`
Object="Table:TwoIndependentVariables" is not in the "new" IDD.
... will be listed as comments on the new output file.
`

Ref to Helpdesk Ticket #15552. Example files were uploaded to the private EnergyPlusDevSupport repo. 

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version)
 - Version of EnergyPlus 9.1 to 9.4
 - Helpdesk ticket number 15552

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by jmarrec** [comment]
@jcyuan2020 I'm not seeing the defect files on https://github.com/NREL/EnergyPlusDevSupport/tree/master/DefectFiles/8000s

**Comment by mjwitte** [comment]
@jmarrec The defect files are there now.

**Comment by jmarrec** [comment]
I see absolutely no problems with the transition of the Defect file. All the values in the 9.2 IDF are perfectly in line with the 9.1 IDF. It's not extremely friendly to match the before and after, but I have taken the time to do it and it appears fine.

Perhaps I am misunderstanding this statement @jcyuan2020 ?

> The transition program will try to use two Table:IndependentVariable objects, a Table:IndependentVariableList, and a Table:Lookup object to replace the original Table:TwoIndependentVariables obeject, **but reporting some values that do not seem to confide with the original table**

Do you mean in the IDF, or do you mean values you get **after** simulation by asking an `Output:Variable`? If not, why do you think the values aren't correct? Is this just a helpdesk ticket that wasn't investigated yet perhaps?

**Comment by jcyuan2020** [comment]
@jmarrec Thanks for looking at this. Back then it was a helpdesk ticket complaining a mismatch on the original and converted fields, and I tried the conversion and saw a few severe errors and a message particularly about this table. Since it was not a full idf that can be tested out if the new conversion would be working in a simulation case, I tried to identify a few values before and after and had originally been convinced there was a mismatch of the x and y coordinates values for the tabulated outputs (a revisit now seems to indicate that I might have misunderstood the newer 9.2 format on this). 
Sorry for the inconvenience--just now I checked a few of the first, the second and the last rows, seems to be ok an aligned table for the new 9.2 version.
